frontend/app/src/App.vue

```diff
@@ -246,5 +248,9 @@ export default class App extends Vue {
       border-bottom: var(--v-rotki-light-grey-darken1) solid thin;
     }
   }
+
+  .v-navigation-drawer {
```

Comment:
Doesn't it work if you nest it under the closing bracket of `&--fixed`?

Reply:
It does!